dags/dags_tests.py

```diff
@@ -14,5 +14,5 @@ def dag_bag(self):
         return DagBag(dag_folder=DAGS_FOLDER, include_examples=False)
 
     def test_no_import_errors(self):
-        assert len(self.dag_bag().dags) == 4
+        assert len(self.dag_bag().dags) == 7
```

Comment:
Dang, is this test even useful? Does it actually catch issues before deployment?

Reply:
I mean, I know I wrote it, and wondering if it is worth keeping still